modules/tensor_mechanics/test/tests/postprocessors/tests

```diff
@@ -0,0 +1,19 @@
+[Tests]
+  [./material_tensor_avg_test]
+    type = 'CSVDiff'
+    input = 'material_tensor_average_test.i'
+    csvdiff = 'material_tensor_average_test_out.csv'
+    requirement = 'The system shall be able to compute the volume average of RankTwoTensor component using MaterialTensorAverage'
+    design = 'MaterialTensorAverage.md'
+    issues = '#15424'
+  [../]
+  [./material_tensor_avg_AD_test]
+    type = 'CSVDiff'
+    input = 'material_tensor_average_test.i'
+    csvdiff = 'material_tensor_average_AD_test_out.csv'
+    requirement = 'The system shall be able to compute the volume average of ADRankTwoTensor component using ADMaterialTensorAverage'
```

Comment:
"The system shall be able to compute the volume average of a rank two tensor component, which was computed using automatic differentiation."